packages/core/lib/std.js

```diff
@@ -73,20 +74,36 @@ module.exports = ({
             "<key> - Unset the variable 'key', and return a value if there is one",
           sleep: "<ms> - Sleep for the specifed amount of milliseconds",
           print: "<...vals> - Print values to stdout",
-          rule: `<name> <body> - Define a new rule 'name'. The 'body' has access to two additional rules, 'bind' and 'return' to take arguments and return a value
-                    bind <...keys> - for each value passed into the rule, store it as a variable using the names passed by 'keys'
-                    return <value> - return the value`,
+          rule: `<name> <body> - Define a new rule 'name'. The 'body' has access to two additional rules, 'bind' and 'return' to take arguments and return a value`,
+          bind: `<...keys> - for each value passed into the rule, store it as a variable using the names passed by 'keys'`,
```

Comment:
this only exists inside the `rule` scope, which is why I had it in `rule`. we should atleast say "can only be used inside the body of a `rule` or something